src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue889.cs

```diff
@@ -12,18 +12,16 @@ public Issue889(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "Assigning to FlyoutPage.Detail after construction doesn't work";
 
-	//[Test]
-	//[Category(UITestCategories.FlyoutPage)]
-	//[Description("Reproduce app crash - Issue #983")]
-	//public void Issue899TestsAppCrashWhenSwitchingTabs()
-	//{
-	//	App.Tap(q => q.Marked("Push new page"));
-	//	App.WaitForElement(q => q.Marked("I have been pushed"));
-	//	App.Screenshot("Push page");
-	//	App.Back();
-	//	App.Screenshot("Navigate back");
+	[Test]
+	[Category(UITestCategories.FlyoutPage)]
+	public void Issue899TestsAppCrashWhenSwitchingTabs()
```

Comment:
This test is failing on Catalyst:
<img width="1170" alt="image" src="https://github.com/user-attachments/assets/7a36836f-9c1c-45c9-aae3-bce457a03208">

Reply:
Resolved and committed the changes